scripts/build.py

```diff
@@ -110,15 +110,19 @@
     'third_party/generated/webfonts/*',
     '*.bundle.js',
     '*.bundle.js.map',
+    'webpack_bundles/*'
+)
+
+PAGES_IN_APP_YAML = (
```

Comment:
Created this in order to simplify the build process a bit. Since previously the `FILEPATHS_NOT_TO_RENAME` was used both as the list of files that shouldn't be renamed and also as a list of pages in app.yaml.